CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+cmake_minimum_required(VERSION 3.11)
+
+set(PRODUCT_NAME cretro)
+string(TOUPPER ${PRODUCT_NAME} MSG_PREFIX)
+
+project(
+    ${PRODUCT_NAME}
+    LANGUAGES C
+)
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
I'm actually not sure of this specifies the C Standard since it's CXX. I'm generally not aware of how CMake handles C/C++ standards

Reply:
Me neither, was asking because I wasn't sure either.